### PR TITLE
Update azure_rm_webappslot document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_webappslot document.

##### ISSUE TYPE

- Docs Pull Request
##### COMPONENT NAME

azure_rm_webappslot
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_webappslot.py [hunk 1]
```diff
@@ -77,8 +77,10 @@
                 description:
                     - Name of the framework.
                     - Supported framework list for Windows web app and Linux web app is different.
-                    - For Windows web app, supported names(June 2018) java, net_framework, php, python, node. Multiple framework can be set at same time.
-                    - For Linux web app, supported names(June 2018) java, ruby, php, dotnetcore, node. Only one framework can be set.
+                    - Windows web apps support C(java), C(net_framework), C(php), C(python), and C(node).
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_webappslot.py]: pls add  support time also

**Review by yungezz — APPROVED** [review]
approve with comments

File: lib/ansible/modules/cloud/azure/azure_rm_webappslot.py (hunk 1 above)
Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_webappslot.py]: Thanks for your review! Already add!

Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_webappslot.py]: Thanks for your review! Already add!

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by Fred-sun** [comment]
@acozine Please take a review this PR when you're free! Thanks a lot!

**Review by samccann — APPROVED** [review]

**Pull request merged**